src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java

```diff
@@ -64,7 +65,7 @@ public LetterStatus getStatus(UUID id, String serviceName) {
     }
 
     @Transactional
-    public UUID send(Letter letter, String serviceName) throws JsonProcessingException {
+    public SendLetterResponse send(Letter letter, String serviceName) throws JsonProcessingException {
```

Comment:
I don't think the service should be even aware of our external responses. The controller is the right place for creating them. Same with the LetterRepository.